packages/web/src/store/selectors/websiteResults.ts

```diff
@@ -56,6 +58,18 @@ const makeSelectScanPackagesByUrl = () =>
         vulnerable: (scanData.vulnerabilities[pkg.name]?.length ?? 0) > 0, // TODO: Drop
         vulnerabilities: scanData.vulnerabilities[pkg.name] ?? [],
         version: semverListAsRange(pkg.versionSet),
+        // TODO: memoize/simplify
+        containingScripts: Array.from(
+          pkg.moduleIds.reduce((acc: Set<string>, id) => {
+            const script = scanResult?.scan?.scanResult?.processedScripts?.find((val) =>
+              val.moduleIds.includes(id)
+            );
+            if (script && script.status === 'processed') {
```

Comment:
Why would a `script.status === 'processed'` check be needed here?

Reply:
I guess it is a type guard

Reply:
Then it would be needed a few lines above too, or am I missing something?